requirements-dev.txt

```diff
@@ -20,8 +20,8 @@ pycdlib==1.13.0
 netifaces==0.11.0
 
 # For tests that validate the produced XUnit output
-xmlschema==1.7.0; python_version < '3.12'
-xmlschema==2.5.0; python_version >= '3.12'
+xmlschema==1.7.0; python_version < '3.13'
```

Comment:
this should be kept to 3.12 to ensure compatibility with python 3.12. see https://github.com/richtja/avocado/commit/b694f497fc212b1f8e5b8daf2c367fcb76372edb